src/assets/scripts/sections/product.js

```diff
@@ -79,6 +83,33 @@ export default register('product', {
         this.updateImages.bind(this),
       );
     }
+
+    this.initImageSwitch();
+  },
+
+  initImageSwitch() {
+    if (!$(selectors.productThumbs).length) {
+      return;
+    }
+
+    $(selectors.productThumbs, this.$container)
```

Comment:
Cache `$(selectors.productThumbs, this.$container)` since you're using it twice

Reply:
Good point!